tests/integration/cloud_governance/common/elasticsearch/test_elasticsearch_operations.py

```diff
@@ -16,16 +17,67 @@ def test_get_upload_update_elasticsearch():
 
     if test_environment_variable['elasticsearch']:
         uuid = str(uuid4())
+        es_index = 'test-cloud-governance-test'
         # verify that data upload to elastic search
         es = ElasticSearchOperations(es_host=test_environment_variable.get('elasticsearch', ''),
                                      es_port=test_environment_variable.get('elasticsearch_port', ''))
         data = {'tool': 'cloud-governance', 'uuid': uuid}
-        es.upload_to_elasticsearch(index='cloud-governance-test', data=data)
-        assert es.verify_elasticsearch_data_uploaded(index='cloud-governance-test', uuid=uuid)
-        id = es.verify_elasticsearch_data_uploaded(index='cloud-governance-test', uuid=uuid)
-        es.update_elasticsearch_index(index='cloud-governance-test', id=id[0], metadata={'cloud-governance_version': '1.0.324'})
-        result = es.get_elasticsearch_index_by_id(index='cloud-governance-test', id=id[0])
+        es.upload_to_elasticsearch(index=es_index, data=data)
+        assert es.verify_elasticsearch_data_uploaded(index=es_index, uuid=uuid)
+        id = es.verify_elasticsearch_data_uploaded(index=es_index, uuid=uuid)
+        es.update_elasticsearch_index(index=es_index, id=id[0],
+                                      metadata={'cloud-governance_version': '1.0.324'})
+        result = es.get_elasticsearch_index_by_id(index=es_index, id=id[0])
         assert result['_source']['uuid'] == uuid
         assert result['_source']['cloud-governance_version'] == '1.0.324'
 
 
+@pytest.mark.skip(reason="No elasticsearch for test")
```

Comment:
I think that we can add [github actions elasticsearch service](https://stackoverflow.com/questions/64204333/service-elasticsearch-is-not-visible-when-run-tests) for testing for integration tests, WDYT ?

Reply:
yes, we can.